docs/docs/deployment/hosting/locally-api.md

```diff
@@ -436,9 +436,19 @@ You can run Flagsmith as a single application/docker container using our unified
 [Docker Hub](https://hub.docker.com/repository/docker/flagsmith/flagsmith) but you can also run the front end as part of
 the Django Application. Steps to do this:
 
-1. `cd frontend; npm run bundledjango`
-2. `cd ../api; python manage.py collectstatic`
-3. `python manage.py runserver`
+```bash
+# Update packages and build django.
+cd frontend
+npm install
+npm run bundledjango
+
+# Copy additional assets with Django
+cd ../api
+python manage.py collectstatic
+
+# Boot the server
+python manage.py runserver
```

Comment:
We have Makefile targets for that. Can we use these>

Reply:
@khvn26 interestingly, we _don't_ have a makefile target for `runserver`. I'm not sure if there was a reason though? 

I tend to agree that for these type of docs we _should_ use the makefile targets where possible but we should perhaps have a discussion (in the next eng meeting perhaps) about the makefiles now that we have onboarded a new engineer.

Reply:
> interestingly, we don't have a makefile target for runserver

We do: `make serve` spins up gunicorn with livereload enabled.

Reply:
If any points are to be made ahead of the discussion, I suggest adding them here: https://github.com/Flagsmith/flagsmith/issues/2893

Reply:
> We do: `make serve` spins up gunicorn with livereload enabled.

I don't think this is the same though, right? `runserver` does other things (e.g. you don't have to set ALLOWED_HOSTS and DEBUG environment variables).

Reply:
Good point! We probably should have those set up for the local environment regardless of which dev server we use.

Reply:
Resolving this in hopes of a later iteration — current docs are correct enough.